### PR TITLE
SeaMonkey support

### PR DESCRIPTION
**Pull request by ruanjiasing**
Issue #7 - Add SeaMonkey support

### CODE REVIEW

File: src/install.rdf [hunk 1]
```diff
@@ -3,7 +3,7 @@
     <Description about="urn:mozilla:install-manifest">
         <!-- General -->
         <em:id>swarth@franklindm</em:id>
-        <em:version>dev-build</em:version>
```
Comment (fofajardo) [inline src/install.rdf:6]: Please leave the version as-is, this is automatically changed during release.

**Review by FranklinDM — CHANGES_REQUESTED** [review]
Also, please follow the `Issue #XXXX - summary` format in the commit message. For this PR, this should be `Issue #7 - Add SeaMonkey support`.

Thanks!

File: src/install.rdf (hunk 1 above)
Comment (3oheicrw) [reply src/install.rdf:6]: Sorry. I don't know what to do. As you can see I uploaded via github web interface. I'm very bad with git. I don't know how to do what you requested.

Comment (3oheicrw) [reply src/install.rdf:6]: What does left the version as-is mean? Does it mean I should just remove this line?

**Pull request closed without merging**